webcompat/static/css/development/components/search-bar.css

```diff
@@ -39,7 +39,12 @@
       font-size: var(--SearchBar-fontSize);
       flex: 1;
       margin: 0 1em;
-      background-color: transparent;
+      background-color: white;
+      padding-top: 5px;
```

Comment:
You can use shorthand padding here. Not sure if we want to use px or em, will let Guillaume comment there.